drivers/i2c/Kconfig.tca9546a

```
@@ -0,0 +1,17 @@
+# Copyright (c) 2020 Innoseis BV
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig I2C_TCA9546A
+	bool "I2C addressable switch"
+	help
+		Enable TCA9546A i2C bus switch
+
+if I2C_TCA9546A
+	config  I2C_TCA9546_ROOT_INIT_PRIO
+	int "TCA9546a root driver init priority"
+	default I2C_INIT_PRIORITY
+
+	config  I2C_TCA9546_CHANNEL_INIT_PRIO
+	int "TCA9546a channel driver init priority"
+	default I2C_INIT_PRIORITY
+endif
```

```suggestion
menuconfig I2C_TCA9546A
	bool "I2C addressable switch"
	help
	  Enable TCA9546A I2C bus switch

if I2C_TCA9546A

config I2C_TCA9546_ROOT_INIT_PRIO
	int "TCA9546a root driver init priority"
	default I2C_INIT_PRIORITY

config I2C_TCA9546_CHANNEL_INIT_PRIO
       int "TCA9546a channel driver init priority"
       default I2C_INIT_PRIORITY

endif # I2C_TCA9546A
```

The whitespace etc. here is very unusual to my eyes Please see above suggestion.